exporters/writers/base_writer.py

```diff
@@ -65,6 +62,15 @@ def _get_compression_format(self):
                                      ''.format(FILE_COMPRESSION.keys()))
         return compression
 
+    def _get_write_buffer(self):
+        kwargs = {
+             'items_per_buffer_write': self.read_option('items_per_buffer_write'),
+             'size_per_buffer_write': self.read_option('size_per_buffer_write'),
+             'items_group_files_handler': self._items_group_files_handler(),
```

Comment:
still the missing part is to load the group files handler associated to the given writer class.

the group files handler class should be indicated as an attribute of the writer class. So, before instantiating the writer, you first load the writer class, get the group files handler class, instantiate it, and then instantiate the writer.